<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="mainlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Virtual Network - Creation</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Name : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>440</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>440</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QLabel" name="dnsLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Do you want to disable the DNS ?</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="dnsCheckBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Yes </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="internalCheckBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>160</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Yes </string>
    </property>
   </widget>
   <widget class="QLabel" name="internalLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>501</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Do you want to make this network &quot;internal&quot; (no external access) ?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="subnetLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="subnetLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>511</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Do you want a specific subnet for your network ? (e.g. 10.88.0.0/16)</string>
    </property>
   </widget>
   <widget class="QLabel" name="subnet2Label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>240</y>
      <width>201</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Leave Empty if you don't want to</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gatewayLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="gatewayLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>511</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Do you want a specific gateway ? (e.g. 10.88.0.1)</string>
    </property>
   </widget>
   <widget class="QLabel" name="gateway2Label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>330</y>
      <width>201</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Leave Empty if you don't want to</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
